collectors/python.d.plugin/mongodb/mongodb.conf

```diff
@@ -80,3 +82,11 @@ local:
     name : 'local'
     host : '127.0.0.1'
     port : 27017
+
+authsample:
+    name : 'secure'
+    host : 'mongodb.example.com'
+    port : 27017
+    authdb : 'admin'
+    user : 'monitor'
+    password : 'supersecret'
```

Comment:
i suggest to comment out the job, otherwise python.d.plugin will create it and try to connect to 'mongodb.example.com'

Reply:
Oops, didn't think about that.